db/migrations/admin-migrations/2023-10-16.0.add-tenants-table.sql

```diff
@@ -13,16 +13,12 @@ END;
 $$ language 'plpgsql';
 -- +migrate StatementEnd
 
-CREATE TYPE email_sender_type AS ENUM ('AWS_EMAIL', 'DRY_RUN');
-CREATE TYPE sms_sender_type AS ENUM ('TWILIO_SMS', 'AWS_SMS', 'DRY_RUN');
 CREATE TYPE tenant_status AS ENUM ('TENANT_CREATED', 'TENANT_PROVISIONED', 'TENANT_ACTIVATED', 'TENANT_DEACTIVATED');
 
 CREATE TABLE tenants
 (
     id VARCHAR(36) PRIMARY KEY DEFAULT public.uuid_generate_v4(),
     name text NOT NULL,
-    email_sender_type email_sender_type DEFAULT 'DRY_RUN'::email_sender_type,
-    sms_sender_type sms_sender_type DEFAULT 'DRY_RUN'::sms_sender_type,
     enable_mfa boolean DEFAULT true,
     enable_recaptcha boolean DEFAULT true,
```

Comment:
Given that we're simplifying the schema, it would be good to remove `enable_mfa` and `enable_recaptcha`. They were dropped in `2024-03-01.alter-tenants-table-drop-enable-mfa-and-recaptcha.sql`